### PR TITLE
Document to add ecosystem test check in releasing task list

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

Ref https://github.com/stylelint/stylelint/issues/7880#issuecomment-2254217924

> Is there anything in the PR that needs further explanation?

See https://github.com/stylelint/stylelint-ecosystem-tester

### CODE REVIEW

**Review by Mouvedia — APPROVED** [review]
Should you update the issue template too?
i.e. l37–42

**Comment by ybiquitous** [comment]
Good catch. I've added a task item for the ecosystem test via 7d5ca13fd. 👍🏼 
I've also adjusted some link texts.

File: docs/maintainer-guide/releases.md [hunk 1]
```diff
@@ -4,19 +4,19 @@
    - include the [template checklist](#new-release-issue-template)
    - if applicable, list any new rules so that we can discuss including them in our configs
 2. If necessary, open a pull request to mark those rules in [`docs/user-guide/rules.md`](../user-guide/rules.md).
-3. Check if [package tests in the ecosystem](https://github.com/stylelint/stylelint-ecosystem-tester#readme) fail with the new version.
-4. If necessary, test `main` locally in the:
-   1. [`stylelint/stylelint-config-recommended`](https://github.com/stylelint/stylelint-config-recommended)
-   2. [`stylelint/stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard)
-   3. [`stylelint/stylelint.io`](https://github.com/stylelint/stylelint.io)
+3. Check if package tests in [stylelint/stylelint-ecosystem-tester](https://github.com/stylelint/stylelint-ecosystem-tester) fail with the new version.
+4. If necessary, test the main branch locally in the:
```
Comment (Mouvedia) [inline docs/maintainer-guide/releases.md:8]:
```suggestion
4. If necessary, test the main branch locally against:
```

File: docs/maintainer-guide/releases.md [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+   1. [stylelint/stylelint-config-recommended](https://github.com/stylelint/stylelint-config-recommended)
+   2. [stylelint/stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard)
+   3. [stylelint/stylelint.io](https://github.com/stylelint/stylelint.io)
 5. Release Stylelint:
    1. If necessary, reorder the changelog entries in the "Prepare x.y.z" pull request so that the widest-reaching changes come first.
    2. Merge the "Prepare x.y.z" pull request.
    3. Open a terminal window in the `stylelint` repository.
    4. Run `npm run release`.
```
Comment (Mouvedia) [inline docs/maintainer-guide/releases.md:16]: note

check https://github.com/softprops/action-gh-release

**Review by Mouvedia — APPROVED** [review]

**Pull request merged**